src/lifecycle/properties-created.js

```diff
@@ -17,6 +17,6 @@ export default function propertiesApply (elem, properties) {
     // Once that bug is fixed, the initial value being passed as the second
     // argument to prop.created() can use the overridden property definition to
     // get the initial value.
-    prop.created && prop.created(elem, initialValue);
+    prop.created(elem, initialValue);
```

Comment:
Does this have a default value now? I wonder why it checked `props.created` before but doesn't now.